baselines/Common Controls Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -1139,3 +1130,38 @@ To configure Supplemental Data Storage per the policy:
 3.	Click the **Supplemental Data Storage** card.
 4.	Ensure the checkbox for "**Russian Federation**" is unchecked.
 6.	Click **Save**.
+
+
+## 16. Additional Google Services
+
+This section covers the Google services that do not have an individual control and whether these services are on or off.
```

Comment:
Do we know which services specifically would be affected by this policy? I looked at the linked resources but couldn't find a concrete list.

Reply:
https://support.google.com/a/answer/7646040?fl=1&sjid=6354300702774302500-NA
https://support.google.com/a/answer/181865?hl=en&fl=1&sjid=6354300702774302500-NA#zippy=%2Cturn-services-on-or-off-for-users
I can't find a concrete list either, is it something we think should be included?

Reply:
Ideally yes, so people can understand the implications of turning these services off. But if Google hasn't published that information, not much we can do.